MobileFilter/sections/specific_web.txt

```diff
@@ -46,6 +46,9 @@ dailynewsonline.jp,yakuji.co.jp,asahi.com,m.winporn.com,slate.com,sankei.com,gam
 !
 careergarden.jp##.p-ad-wrapper
 sankeibiz.jp###textAd_wpics
+yaraon-blog.com##article > div.widgetarea
```

Comment:
Okay but I prefer `###widgetarea_single_undertopimage_sp`. `###widgetarea_underpager_sp` (top page) is anyway needed as not under `article`. Note `###widgetarea_undercommentform_sp` can not be hidden.

Reply:
Please change `yaraon-blog.com##article > div.widgetarea` to `yaraon-blog.com###widgetarea_single_undertopimage_sp` and add `yaraon-blog.com###widgetarea_underpager_sp`.